app/queries/servers/team.ts

```diff
@@ -1,14 +1,44 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {Database} from '@nozbe/watermelondb';
+
+import {Database as DatabaseConstants} from '@constants';
+
 import type ServerDataOperator from '@database/operator/server_data_operator';
+import type TeamChannelHistoryModel from '@typings/database/models/servers/team_channel_history';
+import type TeamModel from '@typings/database/models/servers/team';
+
+const {TEAM, TEAM_CHANNEL_HISTORY} = DatabaseConstants.MM_TABLES.SERVER;
+
+export const addChannelToTeamHistory = async (operator: ServerDataOperator, teamId: string, channelId: string, prepareRecordsOnly = false) => {
```

Comment:
This could've been stored in the System table, but I had an oversight on the existence of the `TeamChannelHistory` so no need to store it in the System table